src/colorizer/types.ts

```diff
@@ -136,6 +136,25 @@ export type DrawSettings = {
   color: Color;
 };
 
+export type VectorConfig = {
+  visible: boolean;
+  key: string;
+  /** Number of time intervals to average over when calculating motion deltas. 5 by default. */
+  timeIntervals: number;
+  color: Color;
+  scaleFactor: number;
+};
+
+export const VECTOR_KEY_MOTION_DELTA = "_motion_";
+
+export const getDefaultVectorConfig = (): VectorConfig => ({
+  visible: false,
+  key: VECTOR_KEY_MOTION_DELTA,
+  timeIntervals: 5,
+  color: new Color(0x000000),
+  scaleFactor: 4,
+});
+
```

Comment:
Move to `constants`?

Reply:
Going to leave this for a follow-up PR because otherwise it'll make the code diff messier